spec/index.bs

```diff
@@ -4265,16 +4265,26 @@ run the following steps:
                 1. Let |textureLayout| be a new {{GPUTextureBindingLayout}}.
 
                 1. If |resource| is a depth texture binding:
-                    1. Set |textureLayout|.{{GPUTextureBindingLayout/sampleType}} to {{GPUTextureSampleType/"depth"}}
-                1. Else if the sampled type of |resource| is `f32`, and |resource| is statically used with a
-                    textureSample* builtin in the shader:
-                    1. Set |textureLayout|.{{GPUTextureBindingLayout/sampleType}} to {{GPUTextureSampleType/"float"}}
-                1. Else if the sampled type of |resource| is `f32`:
-                    1. Set |textureLayout|.{{GPUTextureBindingLayout/sampleType}} to {{GPUTextureSampleType/"unfilterable-float"}}
-                1. Else if the sampled type of |resource| is `i32`:
-                    1. Set |textureLayout|.{{GPUTextureBindingLayout/sampleType}} to {{GPUTextureSampleType/"sint"}}
-                1. Else if the sampled type of |resource| is `u32`:
-                    1. Set |textureLayout|.{{GPUTextureBindingLayout/sampleType}} to {{GPUTextureSampleType/"uint"}}
+                    - Set |textureLayout|.{{GPUTextureBindingLayout/sampleType}} to {{GPUTextureSampleType/"depth"}}
+
+                    Else if the sampled type of |resource| is:
+
+                    -
+                        <dl class=switch>
+                            : `f32`
+                            :: If |resource| is statically used with a textureSample* builtin in the shader:
```

Comment:
bikeshed switches are free-form, they don't have to actually enumerate anything.
So in this case, we should have separate cases for:
  - `f32` if |resource| is statically used with a `textureSample*` builtin in the shader:
  - `f32` otherwise:

Reply:
Done